Changes

```diff
@@ -259,6 +259,9 @@ OCaml 4.14.0
   spurious `as 'a`s in types.
   (Antal Spector-Zabusky, review by Florian Angeletti)
 
+- #10818: Preserve integer literal formatting in type hint.
+  (Leonhard Markert, reviewed by Gabriel Scherer and Florian Angeletti)
+
```

Comment:
The change entry belongs to the "working version" section rather than 4.14 .

Reply:
My understanding is that 4.13.1 is the most recent released version, so if this PR was merged soon, it would be released as part of the next version, and my guess was that this next version would be 4.14.0. That and the fact that the "Working version" section led me to added the changelog entry to the "OCaml 4.14.0" section.

This is guesswork based on how I've seen other projects handle "pending" release notes.

I'm happy to make the change you suggested, but I'd like to understand it. Given that 4.13.1 is the most recent released version, which changelog entries should be added to the "OCaml 4.14.0" section and which changelog entries belong in the (empty) "Working version" section?

Reply:
The set of new features for OCaml 4.14 have already be fixed (in early November). The 4.14 branch is only integrating bug fixes currently. (There should be soon an alpha release for 4.14.0). In other words, all non-bug fixes PRs belong to the working version section. 


Ordinarily, this would have mean that your patch would have been integrated into the development branch and be a part of the 4.15 release.

However, the ordinary release cycle has been suspended for the integration of OCaml multicore into OCaml mainline. Once this integration work is done, we will release OCaml 5.00.0 as a first multicore version of OCaml (around March/April 2022); and only then we will resume the ordinary development cycles.

Consequently, your work will only be merged after the integration of multicore OCaml. And at the earliest, it will be part of the OCaml 5.01 release.

Reply:
Note: I think we should explain the freeze as a note high enough in the README, so that it is visible to people coming to this repository.

Reply:
(This could also be explained in Changes.)

Reply:
Thanks!

+1 for explaining the situation (or adding a link to the explanation) to `Changes`.